### DISCUSSION TITLE
How to use a specific CustomEntry.j2.tex for a given section?

### DISCUSSION THREAD

**Discussion by amin-asdzdh** [rendercv/rendercv#76] [General]
How can I point a custom section (e.g. awards) to use a specific entry type? Currently it seems like the default (OneLineEntry.j2.tex) is used when an unknown section is added.

Thanks for your great work!

**Comment by sinaatalay** [comment]
You can use any of the seven built-in entry types with any section title. Section titles are entirely arbitrary, and they have no default entry types. RenderCV detects which entry type is used by looking at the entry keys.

Unfortunately, there is currently no way to create a `CustomEntry`. However, default entry types can be modified easily.

Let me know if you have any further questions!
